sys/fido2/ctap/ctap_utils.c

```diff
@@ -79,39 +77,43 @@ static void _gpio_cb(void *arg)
     _user_present = true;
 }
 
-void fido2_ctap_utils_led_animation(void)
+void fido2_ctap_utils_wait_for_user_presence(void)
 {
     uint32_t start = ztimer_now(ZTIMER_MSEC);
     uint32_t diff = 0;
     uint32_t delay = 500;
 
     while (!_user_present && diff < CTAP_UP_TIMEOUT) {
-#ifdef LED0_TOGGLE
+#if !IS_ACTIVE(CONFIG_FIDO2_CTAP_DISABLE_LED)
```

Comment:
```suggestion
        if (!IS_ACTIVE(CONFIG_FIDO2_CTAP_DISABLE_LED)) {
```

as per Linux' and thus RIOT's [Coding Convention](https://www.kernel.org/doc/html/latest/process/coding-style.html#conditional-compilation)